File: Homework 1/Statistics HW 1 VS/Statistics HW 1 VS/Form1.vb
Imports System.Diagnostics.Eventing.Reader

Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim fib_number As Long
        fib_number = Convert.ToInt64(TextBox1.Text)
        If fib_number <= 45 Then
            Dim sw As Stopwatch
            sw = New Stopwatch
            sw.Start()
            Label2.Text = "Result : " + fib(fib_number).ToString()
            sw.Stop()
            Label3.Text = "Execution time in ms : " + sw.ElapsedMilliseconds.ToString()
        Else
            Label2.Text = "Result : "
            Label3.Text = "Execution time in ms : "
        End If
    End Sub

    Function fib(value As Long) As Long
        If value = 0 Then
            Return 0
        End If
        If value <= 2 Then
            Return 1
        End If
        Return fib(value - 1) + fib(value - 2)
    End Function
End Class

File: Homework 1/Statistics 1 HW/Statistics 1 HW/Form1.cs

using System.Diagnostics;


namespace Statistics_1_HW
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            long fib_number = long.Parse(textBox1.Text);
            if (fib_number <= 45)
            {
                var stopwatch = new Stopwatch();
                stopwatch.Start();
                label1.Text = "Result : " + fib(fib_number);
                stopwatch.Stop();
                var elapsed_ms = stopwatch.ElapsedMilliseconds;
                label3.Text = "Execution time in ms : "+ elapsed_ms;

            }
            else {
                label1.Text = "Result : ";
                label3.Text = "Execution time in ms : 0";
            }
            
        }

        private long fib(long value) {
            if (value <= 0)
            {
                return 0;
            }
            else if (value <= 2) {
                return 1;
            }else
            {
                return fib(value - 1) + fib(value - 2);
            }
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }

        private void textBox2_TextChanged(object sender, EventArgs e)
        {

        }


    }
}